docs/release-notes/v13.0.0.md

```diff
@@ -5,69 +5,304 @@ sidebar_label: v13.0.0
 image: "img/release-note-link-preview.png"
 ---
 
-_May 30th, 2024_
+_May 09th, 2024_
 
 **Welcome to Mia-Platform v13!**
```

Comment:
```suggestion
:::info
Mia-Platform Console v13.0.0 is **now in Preview!** and will be generally available on May 30th.

Console SaaS users can try out v13.0.0 latest improvements in Preview! Open a Service Request to ask for the creation of a sandbox Company in case you do not have access to any Company.

For self-hosted installations, please read the [following guidelines](#how-to-update-your-console).
:::

**Welcome to Mia-Platform v13!**
```